src/platform/forms-system/test/js/components/Telephone.unit.spec.jsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import sinon from 'sinon';
+
+import Telephone, { CONTACTS } from '../../../src/js/components/Telephone';
+
+describe('Widget <Telephone />', () => {
+  // custom numbers
+  it('should render a number', () => {
```

Comment:
maybe we could add a test to make sure that the `Telephone` renders an `a`, like `expect(wrapper.type()).to.equal('a');`

Reply:
Done